src/components/board/board.tsx

```diff
@@ -1,23 +1,22 @@
 import { Cell } from '../cell/cell';
 import {useState} from 'react';
 
-type boardProps = {
-    board?: Array<string>,
-    onClick?: (cell: number) => void; 
-}
-
-export const Board = (props: boardProps) => {
-    const gameBoard = props.board || ["_","_","_","_","_","_","_","_","_"]
+export const Board = () => {
+    const gameBoard = ["_","_","_","_","_","_","_","_","_"]
 
     const [board, setBoard] = useState<Array<string>>(gameBoard)
     const [playerTokenX, setPlayerTokenX] = useState(true);
     
     
     const changeCell = (i:number) => {
         const updateBoard = [...board];
-        updateBoard[i] = playerTokenX ? "X" : "O";
-        setBoard(updateBoard);
-        setPlayerTokenX(!playerTokenX);
+        if (updateBoard[i] === "_"){
+            updateBoard[i] = playerTokenX ? "X" : "O";
```

Comment:
Not sure if you are still working on validation, but I would expect an error message to pop up if a user clicked on an occupied space.  Something on screen that says, "That space is already occupied, please pick another."  Something like that.  Then it disappears when a valid selection is made.